src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
```

Comment:
Эта переменная не изменяется, поэтому можно воспользоваться const
```suggestion
  const cost = 40;
```